### DISCUSSION TITLE
It is recommended that the numbers in the upper row like as the actual keyboard

### DISCUSSION THREAD

**Discussion by smfmdhs** [obsproject/obs-studio#9603] [Development Questions]
https://lh3.googleusercontent.com/pw/AIL4fc-sVw7bN-gavCcGZqRTZ0RoralIkZ9E8pz9lStjFQ2m8js9fa-LN_WvJWYnP7mjgjgA7-ZuJrWPuUTHln2_RfwOnAhlC2skc5pNX6EeZvsD9vEvYkoHp92fYTiJxIRlDeiEVgrI3ptrtp0gTKGryIeDOAv2m17AewEGmvFUc-srAhPusk2Dx1U91sjejXO0QbZssL5NnwZYrvXjUiQTZUjsPTuljpJtPm6bvInh2xx1ytuth9TjlkDGcC9KxNEYFKVmf-OJd1Hku2VXR9DqNYP8hABxX1S_wSc1oTFdnzyc3gmBcMLCg_jALtXW59kkhfAjjdYxibsldcAXcnYGvPFmi8N31Nl3FqDPixQuOqYKY0fps9YJVwZTtohO4RKh--MWp05tGhzmT654bI1QZSSNgVJ9UxDN26zMmHjYjqPHNxgFSEXyWPjfxpWIP6WvdczaimcIz6PUS9gjCG99g_2ImJG7Av8Aws1z8U3UrFbgwdbPRedZbvf4UOaio1FayZ5e7gUxqdnKRKIMqsxF0SWi7bKUgJQwRMRufIs0ASKUdKjAvtdslO5e-GQzWJ9-zTgZy1nvZC3ywcKsPJtzs3UTjxr5IEBqTNm6Z8U0JdhkXhHlT6U2SADEpb6v8pMe8hiXikLKQg97bMZCWQZrHEZ_81Z3N5gvg3JkbtAHXtgf9e4fFtYzUG3NCv_PQw-CWOYH4gjenyjQP9bKMMsAOwRlBL7DDsrYYPVMZuTc9DR6Ohc3nTpcyPvQLbJEqKxy4cSoYLNIm7IVgi36xv2PcsqVLiubyRkM1rIPaVnXWSs-gRjsD7vFpIbtOT06g0aRvEJeQUHa6MSLbpnLGybalHp6s1Dpmv2RPIc-BzLn4uUfR14wMOB60HgP3E2-NaDjbnh_P_IUb_Kd48u0ipuP-VTEK3Y=w704-h59-s-no?authuser=0

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
